### ISSUE TITLE
Display active conditions on monster info dialog

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#252]
```
When viewing stats on a monster, it should be possible to view what conditions 
are currently active on that particular monster. This list of active conditions 
should be similar to the list that is present on the player overview page.

For example, when using the Chaosreaper and causing the Chaos condition on a 
monster, that condition should be visible when clicking the monster button.
```

Original issue reported on code.google.com by `oskar.wi...@gmail.com` on 12 Oct 2011 at 9:57

**Comment by GoogleCodeExporter** [comment]
```
Should be resolved in r177.
```

Original comment by `oskar.wi...@gmail.com` on 12 Oct 2011 at 10:06
- Changed state: **Ready**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
